app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function BitcoinBasics() {
+    return (
+        <motion.div
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.5 }}
+            className="container mx-auto px-4 py-8"
+        >
+            <Typography variant="h1" className="mb-8 text-4xl border-b border-red-500 pb-2">
+                Bitcoin Basics
+            </Typography>
+            <Typography variant="body" className="mb-4">
+                <br>Bitcoin is kind of a strange thing for people to grasp, mostly because you can’t physically grasp it, but it can still be used to purchase goods and services.</br>
```

Comment:
br tags are self contained line breaks (I prefer to use "br/") and do not have a closing /br

Reply:
That's what I thought I only ever put breaks at the end of a line, let me try that. Thanks again.